Sources/Rendering/Core/RenderWindow/index.js

```diff
@@ -97,7 +97,9 @@ function vtkRenderWindow(publicAPI, model) {
   publicAPI.getStatistics = () => {
     const results = { propCount: 0, invisiblePropCount: 0, gpuMemoryMB: 0 };
     model._views.forEach((v) => {
-      results.gpuMemoryMB += v.getGraphicsMemoryInfo() / 1e6;
+      // TODO : remove test when 'vtkWebGPURenderWindow.getGraphicsMemoryInfo()' is implemented
```

Comment:
Not sure this comment is useful. 
Maybe implementing `getGraphicsMemoryInfo()` would result in an asynchronous call hence this whole function would have to be re-written, not just this test.

Reply:
removed it

Reply:
there is still one comment remaining.

Reply:
oups, bad ctrl-z at the wrong time